src/main/java/org/parabot/environment/scripts/Script.java

```diff
@@ -120,6 +125,7 @@ public final void run() {
                 }
             }
         } catch (Throwable t) {
+            scriptException.handle(t);
```

Comment:
This class is a Runnable, you can also just access the runnable and assign the exception handler to the runnable

Reply:
Moved exception handling code to `ScriptExecuter.java`.